docs/_docs/templates.md

```diff
@@ -6,8 +6,8 @@ permalink: /docs/templates/
 Jekyll uses the [Liquid](https://shopify.github.io/liquid/) templating language to
 process templates. All of the standard Liquid [tags](https://shopify.github.io/liquid/tags/control-flow/) and
 [filters](https://shopify.github.io/liquid/filters/abs/) are
-supported. Jekyll even adds a few handy filters and tags of its own to make
-common tasks easier.
+supported. Jekyll even adds four new tags and even more handy filters of its own to make
```

Comment:
I meant it to be a hint for the reader that this list holds all jekyll tags and filters. I see how the four could easily become outdated when new tags come along. I'm removing the four and rephrasing again to make clearer that this list if 'the' list of jekyll specific tags and filters.